cvat/apps/engine/cloud_provider.py

```diff
@@ -382,10 +383,12 @@ class AzureBlobContainer(_CloudStorage):
     class Effect:
         pass
 
-    def __init__(self, container, account_name, sas_token=None):
+    def __init__(self, container, account_name, sas_token=None, connection_string=None):
```

Comment:
If I understand correctly, specifying `connection_string` is enough to connect to the original container, in this case, account_name should be an optional argument.